LOAN/LoanTypes/MortgageLoans.rdf

```diff
@@ -114,16 +114,53 @@
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-loan-typ-mtg;MortgageBorrower">
 		<rdfs:label xml:lang="en">mortgage borrower</rdfs:label>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-loan-typ-mtg;MortgageIndemnityGuaranty">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-gty;Guaranty"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-dae-gty;isGuaranteedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-typ-mtg;LoanMIGProvider"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-pas-caa;isRealizedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-typ-mtg;MortgageIndemnityInsurancePolicy"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">mortgage indemnity guaranty</rdfs:label>
+		<skos:definition xml:lang="en">Mortgage Indemnity Guaranty (MIG).</skos:definition>
+		<skos:editorialNote xml:lang="en">See notes from SME Review and in MIG Provider. Applies to securitized pool, insures the lender. Additional note (IBM): there is a further application of this. When a lender takes a loan which is a where the value of the loan is greater than 80% of the value of the property, at that point it is required for the lender to also get a private mortgage insurance, so they are paying separately for the mortgage insurance so that if the borrower defaults above 80% then the mortgage insurance pays the loss. In the Loan Party Insurer (new &quot;Party&quot; type) you have Loan Party Insured Ratio (e.g. the 80% in the example above). These are different situations but the same principle. So this needs to be modeled for both. 30 June: Is this Lender or Borrower? since you have one lender and one borrower in a single loan, but multiple lenders in the case of packaging this up for a security - there are then multiple lenders and multiple borrowers. A similar kind of insurance exists in the one lender one borrower scenario i.e. the mortgage loan itself. There are two concepts here. the MIG thing was for bundling these. the MIG might apply across multiple contracts, but still be a fact about &quot;the&quot; contract? The notes about 80% above (IBM) are about the individual loan. ACTION: Tidy this up.</skos:editorialNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-mtg;MortgageIndemnityInsurancePolicy">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-gty;InsurancePolicy"/>
+		<rdfs:label xml:lang="en">mortgage indemnity insurance policy</rdfs:label>
+		<skos:definition xml:lang="en">The physical policy for the Mortgage Indemnity Guaranty.</skos:definition>
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-loan-typ-mtg;MortgageLender">
```

Comment:
Seems suspiciously empty. Is this class even needed?

Reply:
fixed